scripts/quickstart

```diff
@@ -116,6 +122,9 @@ cat >"${CONFIG_DIR}/activate" <(source "${DIR}/make-env")
 echo_bold "==> Adding default compose file..."
 cp "${BASE_DIR}/compose/template.yml" "${CONFIG_DIR}/docker-compose.yml"
 
+echo_bold "==> Creating docker-compose.yml..."
+"${DIR}/compose" config > "${BASE_DIR}/docker-compose.yml"
```

Comment:
This needs to be removed -- as discussed, it's an optional separate action the user can take on their own.

Reply:
Yes, my fault, I thought I had removed it so it must be a slip-up my end :+1: